reference/v8.json

```diff
@@ -163,7 +163,9 @@
     "buffer": {
       "type": "number",
       "default": 128,
-      "doc": "Tile buffer size on each side (measured in 1/512ths of a tile; higher means fewer rendering artifacts near tile edges but slower performance)."
+      "maximum": 512,
```

Comment:
> All tiles are normalized to the same size when they are loaded into GL: 512 "units" wide and tall.

This normalization should be documented somewhere, either as a caveat to `tileSize` or in the documentation string for this property, since the relationship between the two properties is non-obvious.